ui/app/controllers/vault/cluster/init.js

```diff
@@ -48,13 +48,15 @@ export default Controller.extend(DEFAULTS, {
         if (isCloudSeal) {
           data.stored_shares = 1;
           data.recovery_shares = shares;
+          delete data.secret_shares; // API will throw an error if secret_shares is passed for seal types other than shamir (transit, AWSKMS etc.)
```

Comment:
are we getting data from openAPI, if so why is this data on there if the API no longer accepts it?

Reply:
No these properties all map to the form inputs but only some are used based on the seal type.